src/logic/tokens/utils/formatAmount.ts

```diff
@@ -39,3 +39,10 @@ export const formatAmount = (number) => {
 
   return numberFloat
 }
+
+export const formatAmountInUsFormat = (number: string): string => {
+  const options = { style: 'currency', currency: 'USD', minimumFractionDigits: 2, maximumFractionDigits: 8 }
+  const numberFormat = new Intl.NumberFormat('en-US', options)
+  const numberFloat: number = parseFloat(number as string)
+  return numberFormat.format(numberFloat).replace('$', '')
+}
```

Comment:
I'm still in favor of `amount` instead of `number` as variable name

Also, parseFloat shouldn't need the `as string` anymore

Reply:
done